### PR TITLE
Add document with preview1 known issues for dotnet ef

### PR DESCRIPTION
**Pull request by natemcmaster**
This page is the target of the fwlink we want to add in https://github.com/aspnet/EntityFramework/pull/5312.

cc @rowanmiller

### CODE REVIEW

File: docs/cli/dotnet.rst [hunk 1]
```diff
@@ -1,8 +1,161 @@
...
+    {
+        "frameworks": {
+            "net451": {
+                "buildOptions": {
+                    "emitEntryPoint": true
+                }
+            }
+        }
+    }
+
```
Comment (rowanmiller) [inline docs/cli/dotnet.rst]: If their context doesn't have a default constructor they will need to add an implementation of IDbContextFactory, we should include an example for the default context created by the templates (which has a DbContextOptions constructor param).

File: docs/cli/powershell.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package Manager Console (Visual Studio)
 =======================================
```
Comment (rowanmiller) [inline docs/cli/powershell.rst]: The known issues from CLI also apply to PMC commands for ASP.NET Core and .NET Core applications... we should tell folks that for those project types we just invoke the CLI commands, and link to the CLI known issues section.

**Comment by rowanmiller** [comment]
:shipit: once the comments are addressed

File: docs/cli/dotnet.rst (hunk 1 above)
Comment (natemcmaster) [inline docs/cli/dotnet.rst]: Added a small sample of using IDbContextFactory to this PR. See https://github.com/aspnet/EntityFramework.Docs/pull/159/commits/120d3a686beb3d977aa052367c74d348479b00b6

**Pull request merged**